workbook/middlewares/not_found_middleware.py

```diff
@@ -0,0 +1,17 @@
+from django.http import JsonResponse
+from django.contrib import messages
+
+from workbook.middlewares.exception_handler import ExceptionHandler
+
+
+class ObjectDoesNotExistHandler(ExceptionHandler):
+    """Handle uncaught exceptions instead of raising a 404."""
+
+    def handle(self, request, exception):
+        table_name = str(exception.args[0]).split()[0]
+
+        error_message = f"{table_name.capitalize()} not found."
```

Comment:
for the sake of consistency write the functions in the same way as other exceptions